flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/WebMonitorConfig.java

```diff
@@ -78,4 +78,10 @@ public boolean isProgramSubmitEnabled() {
 			ConfigConstants.JOB_MANAGER_WEB_SUBMIT_ENABLED_KEY,
 			ConfigConstants.DEFAULT_JOB_MANAGER_WEB_SUBMIT_ENABLED);
 	}
+
+	public String getAllowOrigin() {
+		return config.getString(
+			ConfigConstants.JOB_MANAGER_WEB_ACCESS_CONTROL_ALLOW_ORIGIN.key(),
```

Comment:
You can simplify this by using `Configuration#getString(ConfigOption<String> option)` instead.